cfme/tests/pod/test_appliance_crud.py

```diff
@@ -219,6 +258,7 @@ def test_crud_pod_appliance(temp_pod_appliance, provider, setup_provider):
 
     Polarion:
         assignee: izapolsk
+        casecomponent: PodAppliance
```

Comment:
I think you want Containers here, but we should confirm with Sudhir. If he wants a different name, I can add it to Polarion and we would need to update https://github.com/ManageIQ/integration_tests/blob/master/conf/polarion_tools.yaml#L94